app/uk/gov/hmrc/voapropertylinking/controllers/ReverseRoutes.scala

```diff
@@ -0,0 +1,314 @@
+// @GENERATOR:play-routes-compiler
```

Comment:
whoa. this generated pile of code shouldn't have been added to git. can you check how this happened?